job-server/test/spark.jobserver/WebApiMainRoutesSpec.scala

```diff
@@ -352,6 +352,9 @@ class WebApiMainRoutesSpec extends WebApiSpec {
       Post("/contexts/meme?num-cpu-cores=3&coarse-mesos-mode=true") ~> sealRoute(routes) ~> check {
         status should be (OK)
       }
+      Post("/contexts/meme?num-cpu-cores=3", "spark.context-settings.test=1") ~> sealRoute(routes) ~> check {
```

Comment:
> Any way to check that test=1 got passed?

So, I hacked the checks into `DummyActor`, which seems... a little weird. Let me know if you're OK with this, or have another idea of how to do it. I'm not super familiar with akka, so I might be missing the easier way to do it.